compiler/tests/success/cmove8.jazz

```diff
@@ -1,51 +1,47 @@
 export
 fn f(reg u8 b, reg u64 x y) -> reg u64 {
   reg u64 r;
-  reg u8 z;
   reg bool c;
 
   r = x;
   c = b == 1;
   r = y if c;
-  r = x if c;
```

Comment:
Why did you remove all these lines?

Reply:
Two reasons: the official one is that these lines bring nothing, they are redundant and do not add anything to the test case; the officious one is that there are some exponential behaviors in the safety checker, and the shorter test case can be checked in some reasonable amount of time.